docker/Dockerfile.tests

```diff
@@ -105,10 +105,10 @@ RUN git clone --branch $PG_AMQP_TAG --depth 1 https://github.com/omniti-labs/pg_
     make install && \
     cd /home/musicbrainz
 
-ENV SOLR_VERSION 7.7.2
+ENV SOLR_VERSION 7.7.3
```

Comment:
Note that we still run 7.7.2 in production. There seems to be no breaking [changes](https://lucene.apache.org/solr/7_7_3/changes/Changes.html) though.

Reply:
I assumed they follow semantic versioning...

Reply:
7.7.2 and 7.7.1 have upgrade notes. Since 7.x is in maintenance only, it might be they don't want to increment minor version either.